day-to-day.md

```diff
@@ -97,11 +93,7 @@ We have also invested in highly-adjustable, ergonomic Steelcase Gesture chairs f
 
 We take coffee pretty seriously (some of us more than others). Coffee beans and other accoutrement are supplied by the company. Feel free to drink any coffee that has been made, and to make more whenever your caffeine-deprived heart desires.
 
-Aside from coffee, we are also serious about stocking water, fizzy water, pop, and beer.
-
-You can track time while drinking, unless you're drinking margaritas with Jeff - that's definitely not billable.
-
-Grocery shopping is typically the responsibility of one person (Jeff). Let that person know what you'd like to have stocked in the fridge.
+Aside from coffee, we are also serious about stocking water, fizzy water, soda, and beer at the office. For remote employees, you should be drinking your own water, fizzy water, soda, and beer.
```

Comment:
💭 We've had different remote policies in the past, and as far as I know we haven't had much internal discussion about changing it. I think it makes sense to chat about it some more now that we're primarily remote due to Covid and the team growing.

```suggestion
Aside from coffee, we like to stock water, fizzy water, soda, and beer at the office. Currently, we don't reimburse non-coffee treats for remote employees, but we'd like to discuss this further as more of the team has shifted to working remotely.
```